src/gg.regression.unity.bots/Tests/Scripts/GameScene/GameSceneTests.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections;
+
+using NUnit.Framework;
+using RegressionGames.StateRecorder;
+using RegressionGames.StateRecorder.Types;
+using RegressionGames.TestFramework;
+using RegressionGames.Types;
+#if UNITY_EDITOR
+using UnityEditor;
+using System.Reflection;
+#endif
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+using Object = UnityEngine.Object;
+// ReSharper disable CheckNamespace
+// ReSharper disable InconsistentNaming
+namespace RegressionGames.Tests.Z_RunMeLast_GameScene
+{
+    [TestFixture]
+
+    public class GameSceneTests
+    {
+        private int oldIndex = -1;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            SavePreviousViewAspect();
+            //force this to match our recording resolution
+            Screen.SetResolution(1920, 1080, false);
+
+            // set to full HD (1920x1080)
+            SetEditorAspectRatio(3);
```

Comment:
Glad you got this to work! Ive been trying this in my CI/CD setup tutorial and was having difficulty getting it working.